pkg/services/accesscontrol/roles.go

```diff
@@ -263,6 +263,19 @@ var (
 			},
 		},
 	}
+
+	instanceConfigWriterRole = RoleDTO{
+		Name:        "fixed:server.config:writer",
+		DisplayName: "Instance config writer",
+		Description: "Read and update Grafana instance configuration.",
+		Group:       "Settings",
+		Permissions: []Permission{
+			{
+				Action: ActionSettingsWrite,
+				Scope:  "settings:auth:oauth_allow_insecure_email_lookup",
+			},
+		},
+	}
```

Comment:
Should this one live in enterprise code instead? Settings api is not available in oss right?

Also I find the name / description a bit strange. The only think this role allows is to update one specific setting

Reply:
I put this role here to be reused by other endpoints in the future and that was the reason that I chose a quite general name, because in the future it can be extended with other permissions. 

Currently it only contains one permission which gives the ability to update only one field. I'm not sure creating a Role just for this would make sense, but regarding the name I think I can scope it better and rename it to something like `fixed:auth.config:writer`? Wdyt?

Reply:
Yeah `fixed:auth.config:writer` is probably better